### PR TITLE
Elastic search 5.4

### PR DESCRIPTION
**Pull request by prashant-cognitree**
feature added csv serializer and avro serializer.

### CODE REVIEW

File: src/main/java/com/cognitree/flume/sink/elasticsearch/AvroSerializer.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+                XContentParser parser = XContentFactory
+                        .xContent(XContentType.JSON)
+                        .createParser(NamedXContentRegistry.EMPTY, data.toString());
+                builder = jsonBuilder().copyCurrentStructure(parser);
+                parser.close();
+            } else {
+                logger.error("Schema File is not configured");
+            }
+        } catch (IOException e) {
+            logger.error("Error in parsing avro format data ", e.getMessage(), e);
```
Comment (Srinathc) [inline src/main/java/com/cognitree/flume/sink/elasticsearch/AvroSerializer.java]: Add a comment to indicate that this exception is logged but not thrown back to continue processing further records.

File: src/main/java/com/cognitree/flume/sink/elasticsearch/CsvSerializer.java [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+ * Created by prashant
+ *
+ * This Serializer assumes the event body to be in CSV format
+ * with custom delimiter specified.
+ */
+public class CsvSerializer implements Serializer {
+
+    private static final Logger logger = LoggerFactory.getLogger(CsvSerializer.class);
+
+    private List<String> names = new ArrayList<String>();
```
Comment (Srinathc) [inline src/main/java/com/cognitree/flume/sink/elasticsearch/CsvSerializer.java]: names and types can be "final"

File: schema.avsc [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (prashant-cognitree) [inline schema.avsc]: file schema.avsc moved to test/resources folder

File: src/main/java/com/cognitree/flume/sink/elasticsearch/AvroSerializer.java (hunk 1 above)
Comment (prashant-cognitree) [inline src/main/java/com/cognitree/flume/sink/elasticsearch/AvroSerializer.java]: change the log statement as per the comment.

File: src/main/java/com/cognitree/flume/sink/elasticsearch/CsvSerializer.java (hunk 2 above)
Comment (prashant-cognitree) [inline src/main/java/com/cognitree/flume/sink/elasticsearch/CsvSerializer.java]: made changes as per the comment.

**Pull request merged**